### DISCUSSION TITLE
Send FT8 Received Signals to PskReporter Website

### DISCUSSION THREAD

**Discussion by aendes** [gdyuldin/x6100_gui#85] [Ideas]
The PSKReporter Website Allows to receive signal reception data.
It would be a useful feature to send the FT8 information directly to pskreporter without using any additional device (just wifi):

Documentation howto send data:
https://pskreporter.info/pskdev.html

**Comment by aendes** [comment]
I am exploring the capabilities of Matt KK5JY's [ft8modem project](https://www.kk5jy.net/ft8modem/), which includes a tool named ft8report. This tool is written in Python and already supports sending data to PSKReporter.

My interest is in running ft8report on the X6100 and enabling it to parse data either from a file or via an open UDP port. Would it be feasible to implement a corresponding sender component to forward data to ft8report?

**Comment by gdyuldin** [comment]
You can parse ft_log.adi for the new records, or check qso_log.db to generate events to ft8report.

**Comment by aendes** [comment]
Thank you for your reply!

From my understanding, the ft_log.adi file only contains finished QSOs. To create a comprehensive tracker for all received data, it would be a fantastic feature to save all received communication events in the same format as WSJT-X does in its ALL.txt file.

This would allow for better analysis and integration of incomplete or observed contacts and upload it to pskreporter. I believe it could be a valuable addition for users who want a more detailed record of their operations.

**Reply by samhiatt** [reply]
I was looking for the same thing. I'd like to be able to automatically report all the stations heard, using just the wifi connection. Bonus points if it could also pull from pskreporter so I could view a list of stations that spotted me. 
Then more bonus points if I can view all of that on a map. 
Then even more bonus points when I can play Doom on it. 😜
